src/main/frontend/src/app/pages/release-graph/release-node.service.spec.ts

```diff
@@ -317,4 +334,212 @@ describe('ReleaseNodeService', () => {
       expect(version).toBeNull();
     });
   });
+
+  describe('Timeline Scale Calculation', () => {
+    // eslint-disable-next-line unicorn/consistent-function-scoping
```

Comment:
I thought this was already turned off and only triggered in extreme cases. A bit strange, but never mind.